src/Graphics/Text/Width.hs

```diff
@@ -14,10 +14,14 @@ module Graphics.Text.Width
 where
 
 import Data.List (foldl')
+import Foreign.C.Types (CInt(CInt), CWchar(CWchar))
 import qualified Data.Text as T
 import qualified Data.Text.Lazy as TL
 
-foreign import ccall unsafe "vty_mk_wcwidth" wcwidth :: Char -> Int
+foreign import ccall unsafe "wcwidth" wcwidth_ :: CWchar -> CInt
```

Comment:
It's safer to prefix the publicly visible names with something less likely to class with any other library that this might be used with. These names have to be unique across all libraries linked into an application which makes the `vty_` prefix attractive.

Reply:
This is a good idea!

Reply:
Especially because it seems like `wcwidth` already exists in general. I pushed a fix on https://github.com/u-quark/vty/commits/fix-for-better-support-for-emoji-width but this PR does not update because it is closed.